jsvpsa/features/stanford_date_formats.feature

```diff
@@ -4,7 +4,7 @@ Feature: Stanford Date Formats
   I want to ensure that the Stanford Date Formats module is working properly
 
 
-  @api
+  @api @live @safe @prod
   Scenario: Stanford Date Formats
     Given the "stanford_date_formats" module is enabled
```

Comment:
This line will enable the module if it is not enabled. So this cannot be marked as @safe.